experimental/calendar/README.md

```diff
@@ -0,0 +1,7 @@
+# icu_calendar [![crates.io](http://meritbadge.herokuapp.com/icu_calendar)](https://crates.io/crates/icu_calendar)
```

Comment:
This was generated by `cargo make generate-readmes`. I'm leaving it since I had this happen a few times now.